### DISCUSSION TITLE
Trouble in Dashboard

### DISCUSSION THREAD

**Discussion by Praczet** [folke/snacks.nvim#139] [Ideas]
Hi,
I would like to know if displaying Trouble todo  in Dasboard is possible now or if it is even a good idea.. that was an intro..

For me I would love to have something like that

Configuration 
- by default  TODO, NOTE, FIX, FIXME  are folded (they could have number as next to file)
- option to choose which one by default unfold (and level)
- optionally pass order. for example: FIX,FIXME,TODO

It could detect if neovim is open in project folder..

![image](https://github.com/user-attachments/assets/66626da8-29c9-4c53-ae22-b3416919a04a)

If it is possible right now I would love to learn how to configure it.

**Comment by folke** [comment]
That's a good idea. Feel free to open a feature request. 
I'll look at it when I have some time.
